src/Compilers/CSharp/Portable/Binder/ForEachLoopBinder.cs

```diff
@@ -1150,7 +1215,7 @@ private bool SatisfiesForEachPattern(ref ForEachEnumeratorInfo.Builder builder,
 
                 if (!lookupResult.IsSingleViable)
                 {
-                    ReportPatternMemberLookupDiagnostics(lookupResult, enumeratorType, CurrentPropertyName, warningsOnly: false, diagnostics: diagnostics);
+                    ReportPatternMemberLookupDiagnostics(lookupResult, enumeratorType, CurrentPropertyName, warningsOnly: false, reportNoSuchMember: true, diagnostics: diagnostics);
```

Comment:
This is another place I'd expect to see `reportedError` adjusted.

Reply:
Same as above. If we want to see both:

error CS0117: 'C.Enumerator' does not contain a definition for 'MoveNext'
error CS0202: foreach requires that the return type 'C.Enumerator' of 'C.GetEnumerator()' must have a suitable public 'MoveNext' method and public 'Current' property

Then we have to not set reportedError to true here.

Reply:
This is the same as above, but for the Current property instead of the MoveNext method